### ISSUE TITLE
Hello, can you not use val when fine-tuning。

### ISSUE THREAD

**Issue by fxlong** [open-mmlab/mmdetection#2825]
What should I do if I do n’t want to use validation set when using faster-rcnn

**Comment by hellock** [comment]
Please refer to the [documentation](https://github.com/open-mmlab/mmdetection/blob/master/docs/getting_started.md#train-with-multiple-gpus). Add `--no-validate` if you want to skip the validation.

**Comment by ryanalexmartin** [comment]
Documentation link above is broken.
